app/models/game.rb

```diff
@@ -14,6 +14,30 @@ def populate!
     populate_right_white_half!
   end
 
+  def white_check?
```

Comment:
Rubocop is angry at this. I had it combined before.

Also, i thought we could use this logic in games_controller by adding

```Ruby
def show
  if white_check?
    flash[:white_check] = "message of check that we can show in the view"
```